x-pack/plugins/actions/server/constants/plugin.ts

```diff
@@ -4,11 +4,11 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { LICENSE_TYPE_BASIC, LicenseType } from '../../../../legacy/common/constants';
+import { LicenseType } from '../../../licensing/server';
 
 export const PLUGIN = {
   ID: 'actions',
-  MINIMUM_LICENSE_REQUIRED: LICENSE_TYPE_BASIC as LicenseType, // TODO: supposed to be changed up on requirements
+  MINIMUM_LICENSE_REQUIRED: 'basic' as LicenseType, // TODO: supposed to be changed up on requirements
```

Comment:
A few plugins (`actions`, `alerts`, `transform`, `watcher`) were still importing the legacy license types, even if this code seems actually dead (couldn't find any usage of these `PLUGIN` constants). Just adapted to use the `licensing` plugin types instead.

Reply:
Do you have any alternative Kibana place for the licensing constants? Or it is OK for each plugin to have the own constants for the licensing?

Reply:
IMHO, As long as the property is correctly typed as `LicenseType`, you probably don't need a constant for it, as TS will ensure you can only use valid strings (`basic`, `gold` and so on) for the prop.